### DISCUSSION TITLE
File sorting

### DISCUSSION THREAD

**Discussion by MarekChyra** [spacedriveapp/spacedrive#1909] [Feature Request]
I'm on Linux and would like to have dot folders first, then folders, dotfiles, and files. Or you can give more customizability that I can for example have some GUI that I can drag a type of file (dot folder, folder, dotfile, file) and move it in what order I want it. It's the only thing I'm not using as my main file manager.

**Comment by utkubakir** [comment]
Thank you for the suggestion, sounds nice. We will explore this in the future!
